addons/rearm/CfgMagazines.hpp

```diff
@@ -0,0 +1,56 @@
+class CfgMagazines {
+    class CA_Magazine;
+    class 60Rnd_CMFlareMagazine : CA_Magazine {
+        displayName = CSTRING(flarelauncher);
+    };
+
+    class VehicleMagazine;
+    class SmokeLauncherMag : VehicleMagazine {
+        displayName = CSTRING(smokelauncher);
+    };
+    class SmokeLauncherMag_boat : VehicleMagazine {
+        displayName = CSTRING(smokelauncher);
+    };
+    
+    class 1000Rnd_Gatling_30mm_Plane_CAS_01_F : VehicleMagazine {
+        displayName = "30mm HEI";
```

Comment:
Cool, for now can you put them in a stringtable file? That way they can be translated properly where needed (and will be easier to move them if that will be decided).